src/Development/IDE/Core/Compile.hs

```diff
@@ -432,7 +434,7 @@ getModSummaryFromBuffer fp contents dflags parsed = do
     , ms_textual_imps = [imp | (False, imp) <- imports]
     , ms_hspp_file    = fp
     , ms_hspp_opts    = dflags
-    , ms_hspp_buf     = Just contents
+    , ms_hspp_buf     = Nothing
```

Comment:
Was this field not used before?

Reply:
Everything works with `Nothing` as far as I can tell. Git grepping in the ghc repo for the field name doesn't show any relevant usage. It really seems fine to drop it